content/blog/2021/05/2021-05-15-configure-plugins-with-jcasc.adoc

```diff
@@ -0,0 +1,115 @@
+---
+:layout: post
+:title: "Configure Plugins with JCasC"
+:tags:
+- jcasc
+- tutorial
+- plugins
+- yaml
+:author: dheerajodha
+description: >
+  This post is a Step-by-Step Guide to configure your plugins with the Jenkins Configuration as a Code (JCasC) plugin  
+opengraph:
+  image: image:/images/post-images/2021-05-15-configure-plugins-with-jcasc/opengraph-image.jpg
```

Comment:
Need to remove the extra `image:` tag and switch the suffix to png
```suggestion
  image: /images/post-images/2021-05-15-configure-plugins-with-jcasc/opengraph-image.png
```

Reply:
Oh, I don't know how I missed that, thank you for pointing it out!